frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-resource.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import {
+  getVmTemplate,
+  getTemplateDisplayName,
+  getOperatingSystemName,
+  getOperatingSystem,
+  getDescription,
+} from 'kubevirt-web-ui-components';
+
+import { ResourceSummary } from '@console/internal/components/utils';
+
+import { DASH } from '@console/shared';
+import { TemplateKind } from '@console/internal/module/k8s';
+
+export const VMTemplateResourceSummary = ({ template }: VMTemplateResourceSummaryProps) => {
+  const base = getVmTemplate(template);
+  const baseLink = base && getTemplateDisplayName(base); // TODO(mlibra): link to a template detail, once implemented
+
+  return (
+    <ResourceSummary resource={template}>
+      <dt>Description</dt>
+      <dd>{getDescription(template)}</dd>
+      <dt>Operating System</dt>
+      <dd>{getOperatingSystemName(template) || getOperatingSystem(template) || DASH}</dd>
```

Comment:
are we going to display the other fields as well?

Reply:
Ahh... yes, sorry, I forgot ... adding :pray:

Reply:
added missing fields, source is a place holder for now, did not find how to get the data volumes yet :blush:

Reply:
can you use the `VMTemplateDetailsFirehose` for that?

Reply:
@suomiy , Thanks :pray: using `firehouse` for that now :+1: 

Do not have data volumes in my okd, so this is not tested ...